### DISCUSSION TITLE
Caffeine: How can i have a script or hotkey to disable swayidle?

### DISCUSSION THREAD

**Discussion by Heus-Sueh** [hyprwm/Hyprland#4726] [Q&A]
I'm playing a game and the swayidle is showing up all the time, how do I disable the swayidle using a hotkey or script?

I saw about a module for the waybar but I didn't understand how to configure it

**Comment by vaxerski** [comment]
`killall swayidle`

**Comment by zakk4223** [comment]
You can also set an idleinhibit window rule. I just set idleinhibit fullscreen for all windows, since 99% of my fullscreen usage is games anyways.

**Reply by Heus-Sueh** [reply]
How do I apply this rule to all fullscreen windows?

**Reply by zakk4223** [reply]
I have 
windowrule = idleinhibit fullscreen, ^.*$

I think you can also do:
windowrulev2 = idleinhibit fullscreen, fullscreen:1
